src/generated/design-tokens-light.css

```diff
@@ -22,11 +22,4 @@
   --card-border-radius: 8px;
   --card-padding: 16px;
   --card-background: #ffffff;
-
-
-  /* Test for RadioButton */
-  --radio-fg: #5D6CD0;
-  --radio-bg: #E8EBFF;
-  --radio-fg-disabled: #C5CDDF;
-  --radio-bg-disabled: #EFF2F9;
```

Comment:
I know this is just paperclips and duct-tape, but I think we should keep it until we get the actual tokens

Reply:
Yes I'll add it back to be sure the component continues working, we just need to be careful about this when doing a `yarn build-tokens`